ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/port_channel_interfaces.schema.yml

```diff
@@ -731,6 +731,69 @@ keys:
             session_tracker:
               type: str
               description: Name of session tracker
+        ip_igmp_host_proxy:
+          type: dict
+          keys:
+            enabled:
+              type: bool
+            groups:
+              type: list
+              primary_key: group
+              # convert_types:
+              #   - dict
+              description: Group Address.
+              items:
+                type: dict
+                keys:
+                  group:
+                    type: str
+                  exclude:
+                    type: list
+                    primary_key: source
+                    # convert_types:
+                    #   - dict
+                    items:
+                      type: dict
+                      keys:
+                        source:
+                          type: str
+                  include:
+                    type: list
+                    primary_key: source
+                    # convert_types:
+                    #   - dict
+                    items:
+                      type: dict
+                      keys:
+                        source:
+                          type: str
+            report_interval:
+              type: int
+              convert_types:
+                - str
+              min: 1
+              max: 31744
+              default: 1
```

Comment:
We should not have defaults in eos_cli_config_gen.
```suggestion
```

Reply:
Done!